### PR TITLE
@task PDA-1418 Add dependency for Protobuf and create .proto file for all messages and methods

### PR DESCRIPTION
**Pull request by dtang-parasoft**

### CODE REVIEW

**Comment by yshi-parasoft** [comment]
Did you upload the correct jar? I downloaded jar from the task and got this:
image

File: src/main/proto/protoServer.proto [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  int32 stock = 1;
+}
+
+message Items {
+  repeated Item items = 1;
+}
+
+message UpdateItemsInStockRequest {
+  int64 id = 1;
+  string operation = 2;
```
Comment (yshi-parasoft) [inline src/main/proto/protoServer.proto:25]: "deduction" and "addition" will be two mutually exclusive fields. So only one of them is required in each request. 
e.g.
{
    id: 1,
    deduction: 1
}
OR
{
    id: 1,
    addition: 1
}

File: build.gradle [hunk 2]
```diff
@@ -169,3 +174,30 @@ bootWar {
 		)
 	}
 }
+
+sourceSets{
+	src {
+		main {
+			java {
+				srcDir 'build/generated/source/proto/main/grpc'
```
Comment (yshi-parasoft) [inline build.gradle:182]: Use "sources" instead of "source", which is consistent with others.

File: src/main/proto/protoServer.proto [hunk 3]
```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package grpc.demoApp;
+
+service protoServer {
```
Comment (tren-parasoft) [inline src/main/proto/protoServer.proto:5]: Service name should be capitalize.

File: src/main/proto/protoServer.proto (hunk 1 above)
Comment (yshi-parasoft) [reply src/main/proto/protoServer.proto]: I have a new idea today which I think might be better. 
Do you think it's better to have an Enumeration type which can be set to "deduction" or "addition, and one more Integer field for value?

Comment (yshi-parasoft) [reply src/main/proto/protoServer.proto]: Also I modified the grammar a little bit in a way which I think better for future extension.
https://github.com/parasoft/parasoft-demo-app/pull/252/commits/dfc1c7fb509ef21f1bcecb59ca59a7e1df5773c7

**Pull request merged**